skits/pipeline.py

```diff
@@ -228,11 +228,11 @@ def predict(self, X, to_scale=False, refit=True, start_idx=0):
                 prediction[:, [idx]] = self.inverse_transform(prediction[:, [idx]])
         return prediction[start_idx:].squeeze()
 
-    def forecast(self, X, start_idx, trans_window=None):
+    def forecast(self, X, start_idx, end_idx=None, trans_window=None):
```

Comment:
I like this idea, but maybe it'd easier for the user to pass in a `forecast_window` rather than an `end_idx`? So, `pipeline.forecast(X, 100, forecast_window=20)` would generate forecasts for points 100 through 120. What do you think?

Reply:
I would be fine with that, I'll make the change!